doc/man3/EVP_MAC.pod

```diff
@@ -47,6 +48,8 @@ EVP_MAC_do_all_provided - EVP MAC routines
  const OSSL_PARAM *EVP_MAC_gettable_params(const EVP_MAC *mac);
  const OSSL_PARAM *EVP_MAC_gettable_ctx_params(const EVP_MAC *mac);
  const OSSL_PARAM *EVP_MAC_settable_ctx_params(const EVP_MAC *mac);
+ const OSSL_PARAM *EVP_MAC_CTX_gettable_params(EVP_MAC_CTX *ctx);
```

Comment:
should the ctx be const for these?

Reply:
I don't believe so.  If a context does have a dynamic selection of gettable/settable parameters, it will need to cache the list once queried.  Thus, the structure's contents could be mutable.